### PR TITLE
security bump: Akka HTTP 10.1.15; Log4j2 2.17.1 (in tests)

### PR DESCRIPTION
**Pull request by ennru**
Bumping some dependencies to bring security fixes in.﻿

### CODE REVIEW

**Comment by ennru** [comment]
Still need to align other Akka HTTP libraries https://app.travis-ci.com/github/akka/alpakka/jobs/552719150#L573

File: project/Dependencies.scala [hunk 1]
```diff
@@ -202,7 +202,7 @@ object Dependencies {
         .map("org.apache.geode" % _ % GeodeVersion) ++
       Seq(
         "com.chuusai" %% "shapeless" % "2.3.3",
-        "org.apache.logging.log4j" % "log4j-to-slf4j" % "2.12.1" % Test
+        "org.apache.logging.log4j" % "log4j-to-slf4j" % "2.16.0" % Test
```
Comment (balagez) [inline project/Dependencies.scala:205]: Right now 2.17.0 is the newest version: https://logging.apache.org/log4j/2.x/index.html

Comment (ennru) [reply project/Dependencies.scala]: Updated.

File: project/Dependencies.scala [hunk 2]
```diff
@@ -202,7 +202,7 @@ object Dependencies {
         .map("org.apache.geode" % _ % GeodeVersion) ++
       Seq(
         "com.chuusai" %% "shapeless" % "2.3.3",
-        "org.apache.logging.log4j" % "log4j-to-slf4j" % "2.12.1" % Test
+        "org.apache.logging.log4j" % "log4j-to-slf4j" % "2.17.0" % Test
```
Comment (ennru) [inline project/Dependencies.scala:205]:
```suggestion
        "org.apache.logging.log4j" % "log4j-to-slf4j" % "2.17.1" % Test
```

**Review by raboof — APPROVED** [review]
Looks reasonable to me!

**Pull request merged**